launcher_go/v2/config/config_test.go

```diff
@@ -0,0 +1,39 @@
+package config_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/discourse/discourse_docker/launcher_go/v2/config"
+	"os"
+	"strings"
+)
+
+var _ = Describe("Config", func() {
+	var testDir string
+	var conf *config.Config
+	BeforeEach(func() {
+		testDir, _ = os.MkdirTemp("", "ddocker-test")
+		conf, _ = config.LoadConfig("../test/containers", "test", true, "../test")
+	})
+	AfterEach(func() {
+		os.RemoveAll(testDir)
+	})
+	It("should be able to load", func() {
```

Comment:
I'm not sure what the convention is like but this doesn't indicate to me which function we are testing against in the module.

Reply:
I was writing them as how it made sense to me under `describe Config`. I've updated it more explicitly now.